### PR TITLE
Feat/illustrated content

### PR DESCRIPTION
**Pull request by vbvnvmfjg**
* [Trello](https://trello.com/c/HZZgI7Vh/124-illustrated-content)
* [Preview in Guide](https://deploy-preview-127--leanwebkit.netlify.com/guide/)
* [Preview on Page](https://deploy-preview-127--leanwebkit.netlify.com/en/test/)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,12 +52,12 @@
...
-    "@storybook/addon-links": "^3.4.7",
-    "@storybook/addon-options": "^3.4.7",
-    "@storybook/addons": "^4.0.0-alpha.24",
-    "@storybook/vue": "^4.0.0-alpha.24",
+    "@storybook/addon-actions": "^4.1.7",
+    "@storybook/addon-knobs": "^4.1.7",
+    "@storybook/addon-links": "^4.1.7",
+    "@storybook/addon-options": "^4.1.7",
+    "@storybook/addons": "^4.1.7",
+    "@storybook/vue": "^4.1.7",
```
Comment (Remcoman) [inline package.json]: did you test storybook with these major version updates?

File: src/client/components/illustrated-content/readme.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Illustrated content
+
```
Comment (Remcoman) [inline src/client/components/illustrated-content/readme.md:2]: echooooooo

shouldn't you put some text here? :-P

Comment (vbvnvmfjg) [reply src/client/components/illustrated-content/readme.md:2]: I updated the text

File: package.json (hunk 1 above)
Comment (vbvnvmfjg) [reply package.json]: No, I haven't. I've reverted the versions.

**Review by Remcoman — APPROVED** [review]

**Pull request merged**